### PR TITLE
Added setTimestamps option

### PR DESCRIPTION
**Pull request by diruuu**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -38,11 +38,20 @@ exports.plugin = {
 
           case 'insertone': {
             const inserted = await db.collection(collection).insertOne(request.payload)
+            if (setTimestamps) {
```
Comment (afgallo) [inline lib/index.js:41]: You are setting this too late as the insert call has already happened. Can you also simplify with defining a variable that holds `new Date().getTime()`

File: package.json [hunk 2]
```diff
@@ -27,7 +27,10 @@
     "eslint": "8.x.x",
     "eslint-config-prettier": "8.x.x",
     "eslint-config-standard": "17.x.x",
+    "eslint-plugin-import": "^2.27.5",
```
Comment (afgallo) [inline package.json]: Please stick to the standard of `.x.x.x`

**Review by afgallo — CHANGES_REQUESTED** [review]
Looking good, thanks @diruuu. Can you also update the documentation on README please

File: lib/index.js (hunk 1 above)
Comment (diruuu) [reply lib/index.js:41]: Fixed this

**Review by afgallo — APPROVED** [review]

**Pull request merged**